src/map/party.c

```diff
@@ -300,6 +300,18 @@ int party_recv_info(const struct party *sp, int char_id)
 			if (i == MAX_PARTY)
 				added[added_count++] = member_id;
 		}
+		
+		ARR_FIND(0, MAX_PARTY, j, p->party.member[j].leader == 1);
+		if (j == MAX_PARTY) {
+			// Leader has changed
+			int k;
+			ARR_FIND(0, MAX_PARTY, k, sp->member[k].leader == 1);
+			if (k < MAX_PARTY) {
+				clif->PartyLeaderChanged(map->id2sd(sp->member[k].account_id), 0, sp->member[k].account_id);
```

Comment:
I think when I wrote it I thought `sp` and `p` could be completely different (so indexes wouldn't match), but this doesn't make much sense when I look at it now.
If I prioritize online members to change leader to, can I be sure that if leader is offline the entire party is offline?